documentation/staging/content/navigate/kubernetes/k8s-ingress-controller.md

```diff
@@ -86,11 +87,19 @@ When editing a route:
 - Use the `Virtual Host` and `Path Expression` fields to define the matching rules that determine which requests match this route.  
 - All requests in the defined rules are routed to the service specified by the `Target Service` field that resides in the namespace specified by the read-only
 `Target Service Namespace` field and the port specified by the `Target Port` field.  
-- To enable TLS between the client
-and the ingress controller, enable the `Enable TLS` option.  Remember, enabling TLS means that the TLS secret
-information must be provided in the `TLS Secret for Ingress Routes` pane. Different ingress controllers support advanced ingress route configuration using ingress controller-specific annotations.  
+- Specify the `Transport Option` for the ingress route:
+    * Select `Plain HTTP` for unencrypted traffic from the client through the ingress controller to the target service.
+    * Select `SSL terminate at ingress controller` for SSL
+      terminating
+      at the ingress controller and then unencrypted traffic from the ingress controller to the target service.  
+      * Enable `Is target service WebLogic Console?` if the target service is the `WebLogic Console` service.  
+    * Select `SSL pass through` for SSL traffic to pass through the ingress
+      controller and then terminate at the target service.  
+      * If you select this option, you must also specify a valid DNS value in 'Virtual Host', and all SSL traffic from 
+        the`Virtual Host` will be routed to the target service.  
+      * Make sure that the `Target Port` supports SSL.
```

Comment:
Capitalize Ingress (several places)

Reply:
No, we do not capitalize ingress. See https://oracle.github.io/weblogic-toolkit-ui/navigate/kubernetes/k8s-ingress-controller/